pandas/core/indexing.py

```diff
@@ -838,8 +838,10 @@ def _getitem_nested_tuple(self, tup: Tuple):
             if self.name != "loc":
                 # This should never be reached, but lets be explicit about it
                 raise ValueError("Too many indices")
-            with suppress(IndexingError):
-                return self._handle_lowerdim_multi_index_axis0(tup)
+            if len(self.obj) > 1 or not any(isinstance(x, slice) for x in tup):
```

Comment:
hmm can we just handle this properly in _handle_lower_multi_index_axis0 ?

Reply:
We could raise an indexingError in there. We can not call xs.

```
            axis = self.axis or 0
            return self._getitem_axis(tup, axis=axis)
```
Would this be preferably to the if condition outside?

Reply:
why is the `len(self.obj)` needed here? that smells a bit

Reply:
Performance reasons, not necessary from a technical standpoint. Will remove it

Reply:
ok pls add a comment then

Reply:
Done